server/server.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const cors = require("cors");
+const server = express();
+server.use(cors());
+server.use(express.json());
+
+const port = process.env.PORT || 3000;
+
+server.get("/", (req, res) => res.send("Hello, world!"));
+server.listen(port, () =>
+  console.log(`Express now departing from http://localhost:${port}!`)
+);
```

Comment:
Needed to bring in your router here and use the server.use method to direct your resource path which looks like it would have "/posts" to the router